src/cmd-cloud-prune

```diff
@@ -114,14 +116,21 @@ def main():
             continue
         duration = convert_duration_to_days(policy[stream][action])
         ref_date = today_date - relativedelta(days=int(duration))
+        pruned_build_ids = []
 
         print(f"Pruning resources of type {action} older than {policy[stream][action]} ({ref_date.date()}) on stream {stream}")
         # Enumerating in reverse to go from the oldest build to the newest one
         for build in reversed(builds):
             build_id = build["id"]
-            if action in build.get("policy-cleanup", []):
-                print(f"Build {build_id} has already had {action} pruning completed")
-                continue
+            if action in build.get("policy-cleanup", {}):
+                if action == "images":
+                    # If images in images-keep in the policy and the images_kept in build doesn't match
+                    # let's run the pruning again on new images and update the images-kept.
+                    if policy.get(stream, {}).get("images-keep", []) != build["policy-cleanup"]["images-kept"]:
```

Comment:
I feel like build["policy-cleanup"]["images-kept"]` could give us an error if that build has never had been processed before? Maybe let's make it easier to read with something like:

```suggestion
                    keep = policy.get(stream, {}).get("images-keep", [])
                    kept = build.get("policy-cleanup", {}).get("images-kept", [])
                    if keep != kept:
```

Reply:
also I don't think we can compare lists this way:

```
>>> ['3','2','1'] != ['1','2','3']
True
```

but these lists are the same, just different order.

Reply:
Fixed now